app/models/outgoing_message.rb

```diff
@@ -305,7 +306,11 @@ def get_body_for_html_display
     text = CGI.escapeHTML(text)
     text = MySociety::Format.make_clickable(text, { :contract => 1, :nofollow => true })
     text.gsub!(/\[(email address|mobile number)\]/, '[<a href="/help/officers#mobiles">\1</a>]')
-    text = ActionController::Base.helpers.simple_format(text)
+
+    # sanitize the text separately to avoid stripping out the link element
```

Comment:
(unless you pass `sanitize: false` to prevent it: https://apidock.com/rails/ActionView/Helpers/TextHelper/simple_format)